core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -946,7 +946,7 @@ public void addEnv(String key, String value) {
     public void addFileSystemBind(final String hostPath, final String containerPath, final BindMode mode, final SelinuxContext selinuxContext) {
 
         final MountableFile mountableFile = MountableFile.forHostPath(hostPath);
-        binds.add(new Bind(mountableFile.getResolvedPath(), new Volume(containerPath), mode.accessMode, selinuxContext.selContext));
+        binds.add(new Bind(mountableFile.getUnixFilesystemPath(), new Volume(containerPath), mode.accessMode, selinuxContext.selContext));
```

Comment:
I am not too sure we should do it here. Future versions of Docker for Windows may become strict about the path (`C:\foo\bar` is good, `/mnt/c/foo/bar` is bad)

I would just ensure that we pass it as `C:\` and not `C:/`

Reply:
It translates into `/c/foo/bar` btw.

Reply:
Ooops, sorry, my memory failed me :) But nevertheless, I would let Docker make the decision on how to translate `C:\foo\bar` into a valid bind